dcat/index.html

```diff
@@ -1482,7 +1494,427 @@ <h4>Property: is referenced by</h4>
 
         </section>
 
+<!-- Versioning properties -->
+
+<section id="Property:resource_previous_version">
+
+<h4>Property: previous version</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a href="http://www.w3.org/ns/dcat#previousVersion">dcat:previousVersion</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>The previous version of a resource in a lineage [[?PAV]].</td>
+</tr>
+<tr>
+<td class="prop">Equivalent property:</td>
+<td><a data-cite="?PAV#d4e459"><code title="http://purl.org/pav/previousVersion">pav:previousVersion</code></a></td>
+</tr>
+<tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?PROV-O#wasRevisionOf"><code title="http://www.w3.org/ns/prov#wasRevisionOf">prov:wasRevisionOf</code></a></td>
+</tr>
+<tr>
+<td class="prop">Usage note:</td>
+<td>
+<p>This property is meant to be used to specify a version chain, consisting of snapshots of a resource.</p>
+<p>The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle.</p>
+</td>
+</tr>
+</tbody>
+</table>
+
+<p>For guidance on the use of this property, see <a href="#version-history"></a>.</p>
+
+</section>
+
+<section id="Property:resource_has_version">
+
+<h4>Property: has version</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a href="http://www.w3.org/ns/dcat#hasVersion">dcat:hasVersion</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>This resource has a more specific, versioned resource [[?PAV]].</td>
```

Comment:
I know this is the definition provided by PAV. However,  I wonder if we can make it more self-explicable. In particular, I am not sure  the more specific versioned resource is defined if we do not mention the "long-living unversioned resource". I would try to explain more saying something like 
```suggestion
<td>This long-living unversioned resource has a more specific versioned resource [[?PAV]].</td>
```

Reply:
Not sure. If we use "long-living unversioned resource" in the definition, we also need to explain what we mean. Actually, this aspect is explained in the usage note. So, if it is not clear, I would expand it there.